_articles/tutorial-download-data-portal.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Download Data Programmatically from Portals
+layout: article
+excerpt: Use R, Python or the command line to download data.
+category: in-practice
+---
+
+Portals allow you to explore data stored in [Synapse](https://sagebionetworks.org/tools_resources/synapse-platform/), a technology platform that allows researchers to aggregate, organize, analyze and share scientific data, code and insights.
+
+All entities in Synapse are automatically assigned a globally unique identifier used for reference with the format `syn12345678`. Often abbreviated to “synID”, the ID of an object never changes, even if the name does. 
+
+Synapse is designed to integrate seamlessly with your analytical workflow. Therefore, options to download data are available in the [R client]({{ site.baseurl }}{% link _articles/tutorial-download-data-portal.md %}#r), [command line client]({{ site.baseurl }}{% link _articles/tutorial-download-data-portal.md %}#command-line) and [Python client]({{ site.baseurl }}{% link _articles/tutorial-download-data-portal.md %}#python).
+
+## Find Files using Explore Data
+
+Search the available files via Explore Data in the navigation bar. The Explore Data section presents several ways to select data files of interest. The top of the page displays pie charts that summarize the number of files based on file annotations of interest, including *Study, Data Type, Assay, Organism, Tissue*, among others. Selection of one of these chart segments will filter the table below to subset the set of files. Alternatively, access the filters using the facet selection boxes to the left of the Data table. For this example, you will [download the *processed* data and *metadata* from the *MC-CAA* study in the Alzheimer's Disease (AD) Knowledge Portal](https://adknowledgeportal.synapse.org/Explore/Data?QueryWrapper0=%7B%22sql%22%3A%22SELECT%20*%20FROM%20syn11346063%22%2C%22limit%22%3A25%2C%22offset%22%3A0%2C%22selectedFacets%22%3A%5B%7B%22concreteType%22%3A%22org.sagebionetworks.repo.model.table.FacetColumnValuesRequest%22%2C%22columnName%22%3A%22study%22%2C%22facetValues%22%3A%5B%22MC-CAA%22%5D%7D%2C%7B%22concreteType%22%3A%22org.sagebionetworks.repo.model.table.FacetColumnValuesRequest%22%2C%22columnName%22%3A%22dataSubtype%22%2C%22facetValues%22%3A%5B%22processed%22%2C%22metadata%22%5D%7D%5D%7D).
```

Comment:
Is this supposed to be specific to AD - related Portals, or for all Portals?  For example, Cancer Complexity doesn't have "Explore Data" nor does it have many of these options for facets.

Reply:
Ideally we could use verbiage that is extensible to all portals. I pushed some new changes.